scripts/apps/search/views/search-filters.html

```diff
@@ -63,7 +63,7 @@
         </ul>
     </div>
 
-    <div sd-toggle-box data-style="circle"  data-title="{{:: filterLabels.categories }}" ng-hide="isEmpty('category')" data-open="true">
+    <div sd-toggle-box data-style="circle"  data-title="{{:: filterLabels.categories || 'CATEGORIES' }}" ng-hide="isEmpty('category')" data-open="true">
```

Comment:
Can you make an if/else branch in [SearchFilters.ts](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/search/directives/SearchFilters.ts#L56) add create defaults there? Don't forget the translations.